app/kumactl/cmd/get/get_dataplanes_test.go

```diff
@@ -30,8 +30,8 @@ import (
 var _ = Describe("kumactl get dataplanes", func() {
 
 	var dataplanes []*mesh_core.DataplaneResource
-
 	BeforeEach(func() {
+
```

Comment:
Can you revert such files (I see a couple of them) with an empty line change to master?
It's easier to review and less potential conflicts